src/app/globals.css

```
@@ -67,3 +67,35 @@
     --red1: rgb(118, 39, 32);
   }
 }
+
+.button {
+  @apply font-bold font-['Pretendard'] text-sm border gap-2 rounded-md;
+}
+
+.button-sm {
+  @apply p-2;
+}
+
+.button-md {
+  @apply px-4 py-2.5;
+}
+
+.button-lg {
+  @apply w-[109px];
```

```suggestion
  @apply w-full;
```
large 타입 버튼은 영역을 꽉 채우는 스타일로 수정 부탁드립니다! 🙇‍♀️

---

적용했습니다!